src/users.js

```diff
@@ -5,12 +5,16 @@ const USERS_GET =
   "SELECT id, first_name, last_name, username, title, admin FROM users ORDER BY last_name";
 const USER_GET =
   "SELECT first_name, last_name, username, admin FROM users WHERE id = ?";
+const USER_ID_PHONE_GET = "SELECT phone_number FROM contacts WHERE user_id = ?";
 const PASSWORD_UPDATE = "UPDATE users SET password = ? WHERE id = ?";
 
 exports.getUsers = async (pool) => execQuery(pool, USERS_GET, null);
 
 exports.getUser = async (pool, user) => execQuery(pool, USER_GET, user);
 
+exports.getUserIdPhone = async (pool, user) =>
```

Comment:
Why not just call this getPhoneNumber and instead of calling the param user call it userId so I know that I don't pass the whole user object in.

Reply:
Valid question—I wanted to be a little more explicit because it's not a contact's phone number like `getPhoneNumber()` implies. It's a _user_'s phone number